app/_src/mesh/guides/jwt-validation.md

```diff
@@ -0,0 +1,152 @@
+
+## Use an OPA policy to validate JWT tokens
+
+{% navtabs %}
+{% navtab Kubernetes %}
+1. Create the `require-jwt` OPA policy that enforces JWT validation when the `frontend` service makes a request to the `backend` service in the `kuma-demo` namespace:
+
+```bash
+echo "
+apiVersion: kuma.io/v1alpha1
+kind: OPAPolicy
+mesh: default
+metadata:
+  name: require-jwt
+  namespace: kuma-demo
+spec:
+  selectors:
+  - match:
+      kuma.io/service: '*'
+  conf:
+    policies:
+      - inlineString: |
+          package envoy.authz
+
+          import input.attributes.request.http as http_request
+
+          default allow = false
+
+          token = {\"valid\": valid, \"payload\": payload} {
+              [_, encoded] := split(http_request.headers.authorization, \" \")
+              [valid, _, payload] := io.jwt.decode_verify(encoded, {\"secret\": \"secret\"})
+          }
+
+          allow {
+              is_token_valid
+              action_allowed
+          }
+
+          is_token_valid {
+            token.valid
+            now := time.now_ns() / 1000000000
+            token.payload.nbf <= now
+            now < token.payload.exp
+          }
+
+          action_allowed {
+            http_request.method == \"GET\"
+            token.payload.role == \"admin\"
+          }
+" | kubectl apply -f -
+```
+1. Export the token to a variable and make the request (somehow not manually, like how would this be done in real life):
+{% endnavtab %}
+{% navtab Universal %}
+1. Create the `require-jwt` OPA policy that enforces JWT validation when the `frontend` service makes a request to the `backend` service in the `kuma-demo` namespace:
+
+```bash
+echo "
+type: OPAPolicy
+mesh: default
+metadata:
+  name: require-jwt
+  namespace: kuma-demo
+selectors:
+- match:
+    kuma.io/service: '*'
+conf:
+  agentConfig: # optional
+    inlineString: | # one of: inlineString, secret
+      decision_logs:
+        console: true
+  policies: # optional
+    - inlineString: | # one of: inlineString, secret
+        package envoy.authz
+
+        import input.attributes.request.http as http_request
+
+        default allow = false
+
+        token = {"valid": valid, "payload": payload} {
+            [_, encoded] := split(http_request.headers.authorization, " ")
+            [valid, _, payload] := io.jwt.decode_verify(encoded, {"secret": "secret"})
+        }
+
+        allow {
+            is_token_valid
+            action_allowed
+        }
+
+        is_token_valid {
+          token.valid
+          now := time.now_ns() / 1000000000
+          token.payload.nbf <= now
+          now < token.payload.exp
+        }
+
+        action_allowed {
+          http_request.method == "GET"
+          token.payload.role == "admin"
+        }
+" | kumactl apply -f -
+```
+1. Export the token to a variable and make the request (somehow not manually, like how would this be done in real life):
+{% endnavtab %}
+{% endnavtabs %}
+
+## Test the JWT validation configuration
```

Comment:
@lahabana Since I was thinking this should be more of a straightforward reference/how to doc instead of a tutorial, I think that a validation/testing section probably isn't necessary, unless we want to provide general testing guidance for how they could test JWT validation with OPA in their environment.